packages/editor/src/browser/editor-widget.ts

```diff
@@ -7,9 +7,11 @@
 
 import { SelectionService } from '@theia/core/lib/common';
 import { Widget, BaseWidget, Message, Saveable, SaveableSource } from '@theia/core/lib/browser';
```

Comment:
`Navigatable` should be imported from `@theia/core/lib/browser`

Reply:
ok